From 20b72e82fca4c3d81eca2f048090123addb845ed Mon Sep 17 00:00:00 2001
From: Duy Dang <duy.dang.yb@renesas.com>
Date: Wed, 22 Jul 2020 11:11:16 +0700
Subject: [PATCH 248/504] ARM: dts: r8a7743: Add tddmac support to SoC dtsi

This patch adds Image Extraction Direct Memory Access Controller
(tddmac) module to R8A7743 (RZ/G1M) device tree.

Signed-off-by: Duy Dang <duy.dang.yb@renesas.com>
Signed-off-by: Hao Bui <hao.bui.yg@renesas.com>
---
 arch/arm/boot/dts/r8a7743.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/r8a7743.dtsi b/arch/arm/boot/dts/r8a7743.dtsi
index 5220b07..66dafae 100644
--- a/arch/arm/boot/dts/r8a7743.dtsi
+++ b/arch/arm/boot/dts/r8a7743.dtsi
@@ -413,6 +413,14 @@
 			dma-channels = <15>;
 		};
 
+		tddmac@0xfea00000 {
+			compatible = "renesas,vspm-tddmac","renesas,vspm";
+			reg = <0 0xfea00000 0 0x200>;
+			interrupts = <0 285 IRQ_TYPE_LEVEL_HIGH>;
+			clocks = <&mstp1_clks R8A7743_CLK_2DDMAC>;
+			power-domains = <&cpg_clocks>;
+		};
+
 		audma0: dma-controller@ec700000 {
 			compatible = "renesas,dmac-r8a7743",
 				     "renesas,rcar-dmac";
-- 
2.7.4

